legal-api/src/legal_api/models/filing.py

```diff
@@ -133,7 +133,17 @@ class Source(Enum):
                 'ULC': 'BCINC',
                 'CC': 'BCINC',
                 'CP': 'OTINC'
-            }
+            },
+            'temperoryCorpTypeCode': 'TMP'
```

Comment:
temporary is spelled wrong
What is this for? We currently have a bootstrap for businesses until they are created, is this for a new flow?

Reply:
Introduced here since this code is specific for incorporationApplication. We created a new ticket #10737 to support new temporary code for registration filing

Reply:
i think you can still use the legalType, I don't think we need another temp type.
we could create another range in the bootstrap if needed.